src/paginationCalls/pageDescriptor.ts

```diff
@@ -103,6 +113,102 @@ export class PageDescriptor implements Descriptor {
     return stream;
   }
 
+  // create async iterator(settings) => iterable
+  async(apiCall: GaxCall, request: {}, options: CallSettings): {} {
```

Comment:
`async` is a method name here? Is it allowed? :)   Can we rename it?

Also, for the return type, does `@types/node` define anything better than `{}` for an iterable?

Reply:
`request` should be a `RequestType` (we have it defined in `apitypes.ts`).

Reply:
1. It is allowed but yes we can rename it, since we call it `pagedExpandAsync` and the implementation from micro-generator side is `this._descriptor.page.pagedExpand.async[or other names]` I thought `async` may imply the way of using async iterator. 

2. I tried `Object` and it suggested me to use {} instead, but I can look up other types

Reply:
Sure, overlooked it, changed the request type, thanks!

Reply:
`asyncIterate` is used for calling async iterator, avoid the misunderstanding of word ``async

Reply:
Yeah, `async` is a reserved word so anything else (e.g. `asyncIterate`) is fine.

Reply:
Sounds like the type is `Iterator<T>` where `T` is the actual element type. https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html

Reply:
Thank you!